OpenTelemetry.Exporter.ZPages/OpenTelemetry.Exporter.ZPages.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <Description>ZPages exporter for OpenTelemetry</Description>
+    <PackageTags>$(PackageTags);ZPages;distributed-tracing</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Implementation\" />
```

Comment:
Don't think we need this as there isn't an implementation folder unless you have one locally you're trying to prevent being put into git?

Reply:
You're right, I have updated the csproj